.travis.yml

```diff
@@ -1,29 +1,35 @@
 language: python
-python: 2.7
 sudo: false
-# TODO(issue 532): Fix syntax when 3.5 is natively available upstream
+
 matrix:
   include:
-    - python: 3.5
-      env:
-      - TOX_ENV=py35
+  - python: 2.7
+    env: TOX_ENV=flake8
+  - python: 2.7
+    env: TOX_ENV=docs
+  - python: 2.7
+    env: TOX_ENV=gae
+  - python: 2.7
+    env: TOX_ENV=py27
+  - python: 3.3
+    env: TOX_ENV=py33
+  - python: 3.4
+    env: TOX_ENV=py34
+  - python: 3.5
+    env: TOX_ENV=py35
+  - python: pypy
+    env: TOX_ENV=pypy
+  - python: 2.7
+    env: TOX_ENV=system-tests
+  - python: 3.4
+    env: TOX_ENV=system-tests3
```

Comment:
Sorry for the late comment.  What do you think using something like [tox-travis](https://pypi.python.org/pypi/tox-travis) to simplify this part.